src/components/Home.js

```diff
@@ -10,46 +14,40 @@ const Home = () => {
     dispatch(getDoctors());
   }, []);
 
+  const social = [
+    { icon: fb },
+    { icon: twitter },
+    { icon: insta },
+  ];
+
   return (
     <>
```

Comment:
- [ ] It would be great to use the same width for the doctor cards

![Screenshot_20220827_132147](https://user-images.githubusercontent.com/25227022/187030109-a79cd025-cefd-4c5d-a5e4-98872f6c78d4.png)